Example/Example.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\ViewGenerator.1.0.293\build\ViewGenerator.props" Condition="Exists('..\packages\ViewGenerator.1.0.293\build\ViewGenerator.props')" />
-  <Import Project="..\packages\ViewPacker.1.0.12\build\ViewPacker.props" Condition="Exists('..\packages\ViewPacker.1.0.12\build\ViewPacker.props')" />
+  <Import Project="..\packages\ViewGenerator.1.0.296\build\ViewGenerator.props" Condition="Exists('..\packages\ViewGenerator.1.0.296\build\ViewGenerator.props')" />
```

Comment:
Actually, you should revert these upgrades (of ViewGenerator and ViewPacker) in Example. The azure deployment needs to generate the packages first, only then we can upgrade

Reply:
In packages.config as well